layer/layer.go

```diff
@@ -168,6 +177,7 @@ type MountInit func(root string) error
 // read-only and read-write layers.
 type Store interface {
 	Register(io.Reader, ChainID) (Layer, error)
```

Comment:
fine with me, I guess the linked PR should be updated as well not to clutter this interface